variables.tf

```
@@ -8,8 +13,89 @@ variable "location" {
   type        = string
 }
 
+variable "log_analytics_workspace_id" {
+  description = "The ID of the Log Analytics workspace to send diagnostics to."
+  type        = string
+}
+
+variable "administrator_login" {
+  description = "The login username of the administrator for this MySQL server."
+  type        = string
+}
+
+variable "databases" {
+  description = "A list of databases to create on the MySQL Flexible Server."
+  type = map(object({
+    name      = string
+    collation = optional(string, "utf8_general_ci")
+    charset   = optional(string, "utf8")
+  }))
+}
+
+variable "server_version" {
```

Could shorten this according to naming convention (unless you think the longer name is more descriptive?):

```suggestion
variable "version" {
```

---

The variable name "version" is reserved for module version

ex
```
module "mysql" {
  source = "equinor/mysql/azurerm"
  version = "v0.1.0"
```

---

ahhh, of course! I think `server_version` is perfect then 🙌